MEP_Engine/Compute/Mechanical/WaterSide/ProcessSensibleHeat.cs

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+using BH.oM.Base.Attributes;
+using BH.oM.Quantities.Attributes;
+
+namespace BH.Engine.MEP.Mechanical
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Calculates the change in heat during a process for water given volumetric flow rate and two temperature points." +
+            "Per ASHRAE Handbook HVAC Systems and Equipment 2020 (SI Edition) Chapter 13 Hydronic Heating and Cooling.")]
```

Comment:
You'll need some form of space between the lines as otherwise when this renders to the user it'll just read as `two temperature points.Per ASHRAE Handbook`.

Given that ASHRAE is the American Society of Heating, Refrigerating, and Air-Conditioning Engineers, is this the right place for a societal agnostic calculation to be derived from?

Reply:
ASHRAE is a very common resource and commonly is cited within CIBSE and ISHRAE. I am comfortable using it as a region agnostic source given the equations used in these methods are fundamental's equations.